src/assets/scripts/client/trafficGenerator/buildPreSpawnAircraft.js

```diff
@@ -101,21 +97,45 @@ const _calculateDistancesAlongRoute = (waypointModelList, airport) => {
             continue;
         }
 
-        if (isWithinAirspace(airport, waypointModel.relativePosition)) {
-            distanceFromClosestFixToAirspaceBoundary = nm(calculateDistanceToBoundary(airport, waypointModel.relativePosition));
-            totalDistance += distanceFromClosestFixToAirspaceBoundary;
+        const distanceBetweenWaypoints = distance2d(previousWaypoint.relativePosition, waypointModel.relativePosition);
```

Comment:
I knew I should have finished that damn spawn method fixing PR.... these will be not so fun to resolve conflicts on... but hey, that's what I get! 😢